flake.nix

```diff
@@ -1,9 +1,6 @@
 {
   inputs = {
-    flake-compat = {
-      url = "github:edolstra/flake-compat";
-      flake = false;
-    };
+    flake-compat.url = "github:edolstra/flake-compat/refs/pull/65/head";
```

Comment:
i don't know whether edolstra/flake-compat#64 affects us, but this patch shouldn't cause harm